src/server.js

```diff
@@ -1,20 +1,25 @@
 import express from 'express';
 import * as dotenv from 'dotenv';
 import cors from 'cors';
-
+import bodyParser from 'body-parser';
```

Comment:
body-parser might be deprecated. Maybe you can use app.use(express.json()) instead.

Reply:
Replaced